pandas/core/groupby.py

```diff
@@ -997,16 +997,16 @@ def tail(self, n=5):
         Examples
         --------
 
-        >>> df = DataFrame([[1, 2], [1, 4], [5, 6]],
+        >>> df = DataFrame([['a', 1], ['a', 2], ['b', 1], ['b', 2]],
                             columns=['A', 'B'])
         >>> df.groupby('A', as_index=False).tail(1)
```

Comment:
I get a different index than your result:

```python
In [9]: df.groupby("A", as_index=False).tail(1)
Out[9]:
   A  B
1  a  2
3  b  2

In [10]: df.groupby("B", as_index=False).head(1)
Out[10]:
   A  B
0  a  1
1  a  2
```

Is that a copy-paste error, or a bug?